.github/workflows/ci.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         run: cd esp-hal-smartled/ && cargo +esp check --features=esp32s3
       # Ensure documentation can be built (requires a chip feature!)
       - name: rustdoc
-        run: cd esp-hal-smartled/ && cargo doc --features=esp32c3
+        run: cd esp-hal-smartled/ && cargo doc --features=esp32c3,esp-hal-common/eh1
```

Comment:
I thought in your other PR we determined this wasn't needed? Or am I missing something here?

Reply:
No, no, this is needed so that the eh1-specific links don't throw warnings. It was just in the wrong place in that PR.

Reply:
Oh I see how my comment was misleading. A specific example of a warning fixed by the feature can be seen [here](https://github.com/esp-rs/esp-hal/actions/runs/5646002282/job/15292979037#step:12:316)